config/travis/deploy_github_pages.sh

```diff
@@ -13,8 +13,8 @@ set -o nounset # exit if variable is unset
 cd build/docs/html5
 
 git init
-git config user.name 'Deployment Bot (Travis)'
```

Comment:
Changed the wrong files when using IntelliJ to change all occurrences.

Reply:
fixed!